setup.cfg

```diff
@@ -9,7 +9,7 @@ tag = True
 
 [flake8]
 ignore = E241,E266,W503
-max-line-length = 82
+max-line-length = 84
```

Comment:
We can also go for 88 here, according to [Black documentation](https://black.readthedocs.io/en/stable/the_black_code_style.html#line-length) 😄

Reply:
I think that's too long in general. Well actually what you would really want is between 82 and 88, so basically some penality increasing from 82 to 88, but flake8 does not understand that. I don't know people tend to go to longer lines these days. Maybe we should make it 88

Reply:
I can format some files with `black` to play around with this and see how the code looks. It does not need to be decided right now. Just found it interesting that longer lines are being used nowadays.

Reply:
If you want, sure. Just make a PR so we can see.